recipes/ipyopt/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "ipyopt" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a105ef68227304c0bb81c7583cf9ea83d27bfc3584a7a2e531fdef991d49e03
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+  host:
+    - python
+    - ipopt
+    - numpy 1.14
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('ipopt') }}
+    - numpy
```

are you compiling agains ``numpy``? if so, please change it to:
```suggestion
    - {{ pin_compatible('numpy') }}
```

---

👍 I expected numpy to have run_exports!